test/poseidon.test.ts

```diff
@@ -2,7 +2,7 @@ import BN from "bn.js";
 import { Field } from "../src/field";
 import { Poseidon } from "../src/poseidon";
 import { config } from "./config-basic";
```

Comment:
is config-basic a new file?

Reply:
no config-basic is use because of I don't know the right config at first, is just a test.